### PR TITLE
move create_app() out of __init__.py

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Moving create_app() out of __init__.py means that required settings don't need to be passed when running most unit tests.

This make testing individual test in PyCharm a lot easier

### How to review 
Test everything works as normal. Check that unit tests that do no start the app don't require the mandatory settings to be set.

### CODE REVIEW

File: app/setup.py [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+
+def get_minimized_asset(filename):
+    """
+    If we're in production and it's a js or css file, return the minified version.
+    :param filename: the original filename
+    :return: the new file name will be .min.css or .min.js
+    """
+    if settings.EQ_MINIMIZE_ASSETS:
+        if 'css' in filename:
```
Comment (collisdigital) [inline app/setup.py:262]: This looks dangerous, shouldn't it use `endswith`?

Comment (jonnyshaw89) [reply app/setup.py:262]: Probably, but EQ_MINIMIZE_ASSETS doesn't actually work anyway. and isn't really the context of this change.

Comment (collisdigital) [reply app/setup.py:262]: Add at tech debt card to either remove or fix this then?

Comment (jonnyshaw89) [reply app/setup.py:262]: https://trello.com/c/jK5F7hgr/432-enable-asset-minification-by-default ?

Comment (collisdigital) [reply app/setup.py:262]: :+1: I added some more text to that card

**Review by ajmaddaford — APPROVED** [review]

**Review by dcdarrell9 — APPROVED** [review]

**Pull request merged**